scripts/oem/set-guarded-fru-leds.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+# This shell script sets operational status to false for guarded FRUs.
+
+# Skip running this script if the chassis is powered ON.
+current_chassis_status=$(busctl get-property xyz.openbmc_project.State.Chassis \
+        /xyz/openbmc_project/state/chassis0 xyz.openbmc_project.State.Chassis \
+    CurrentPowerState | cut -d" " -f2)
+
+if [ "${current_chassis_status}" = \
+"\"xyz.openbmc_project.State.Chassis.PowerState.On\"" ]; then
+    echo "Current chassis power state is , $current_chassis_status . \
+    Exit set-guarded-fru-leds.sh \
+    script successfully without setting operational status for guarded FRUs."
+    exit 0
+fi
+
+#Extract list of DIMMs and Cores that has been guarded. This is required to set
```

Comment:
Extract list of DIMMs and Cores that has been guarded
Extract list of FRUs that has been guarded

Reply:
Are there any other FRUs that can be guarded?